integration/mock-backend-server/src/main/java/org/wso2/choreo/connect/mockbackend/MockBackendSOAP.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.choreo.connect.mockbackend;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import org.slf4j.LoggerFactory;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.equalToXml;
+import static com.github.tomakehurst.wiremock.client.WireMock.containing;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.options;
+
+public class MockBackendSOAP extends Thread{
+    private static final org.slf4j.Logger log = LoggerFactory.getLogger(MockBackendSOAP.class);
```

Comment:
Have a memory that this logger not working for mockbackend and having to use `private static final Logger log = LogManager.getLogger(MockBackendSOAP.class);` Shall we double check?

Reply:
fixed in https://github.com/wso2/product-microgateway/pull/2972/commits/5c91048f9fa451e4b2f2d91dac704d3f6b755cd3

Reply:
Does slf4j LoggerFactory work for mockbackend? if not we will not be able see the logs from this server.

Reply:
Sorry for the confusion with the above comment..

I have tested running mockbackend separately and could be able to see these log statements.. So I think slf4j LoggerFactory works for mockbackend.
![image](https://user-images.githubusercontent.com/17524063/185173733-59feced9-8ce7-4a24-b4f9-85314d504df4.png)

Reply:
Great! For some reason it didn't work for me. So I concluded that it doesn't work for the mock backend and even made many updates just to fix the logs there. This is great news. Thanks!